packages/v4/src/content/design-guidelines/components/form/forms.md

```diff
@@ -113,6 +113,41 @@ Form submit buttons should be disabled until a user starts filling out the form
 * For a submitted form, disable the submit button while a user's responses are processing to avoid a duplicate submission.
 * For a processing form, use feedback messages and progress indicators like spinners or progress bars if the processing time might exceed user expectations.
 
+## Form sections
+Form sections sort form fields into non-hierarchical groups based on common information type, step, or visual preference. PatternFly form sections support a header followed by a left-aligned list of unnested and uncollapsible form groups, so they create a relatively flat information hierarchy. Form sections are ideal for breaking lengthy forms into simple, evenly spaced segments with all information exposed. 
+
+Default to grouping data with form sections, especially if all form fields are required. If you need to visualize complex or nested information, use [field groups](#field-groups) instead.
+
+To distinguish between form sections, title them based on the reason you grouped their inputs together. You may forego form section titles, but make sure visual separation isn't incredibly meaningful to your content. To keep form sections accessible, add an `aria-label` to indicate why you're sectioning each set of form fields together.
```

Comment:
I would re-word to clarify this a little.  Something like "You may forego from section titles and solely rely on the added spacing between sections, but make sure visual separation isn't incredibly meaningful to your content. If you do decide to forego the form section titles, add an 'aria'label' to indicate ..... etc"

Although if I were to read "but make sure visual separation isn't incredibly meaningful to your content" i'm not sure how I would interpret it. Wondering if you could reword to make it more clear - like are you saying that you shouldn't have a blank space if it's going to confused the user as to why it's there?